### PR TITLE
Update tests with Ragger + CI.

### PR DESCRIPTION
**Pull request by agrojean-ledger**

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pytest
+ragger[speculos]
+ragger[ledgerwallet]
+protobuf==3.20.3
```
Comment (fbeutin-ledger) [inline tests/requirements.txt]: Not needed

File: tests/fantom_client/utils.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            for p in splitted_path]
+
+
+def packed_bip32_path_from_string(path: str) -> bytes:
+    bip32_paths = bip32_path_from_string(path)
+    
+    return b"".join([
+            len(bip32_paths).to_bytes(1, byteorder="big"),
+            *bip32_paths
+        ])
```
Comment (fbeutin-ledger) [inline tests/fantom_client/utils.py]: Take the equivalent function from `ragger.bip`

**Review by fbeutin-ledger — DISMISSED** [review]

File: tests/fantom_client/utils.py (hunk 2 above)
Comment (agrojean-ledger) [reply tests/fantom_client/utils.py]: Done

File: tests/requirements.txt (hunk 1 above)
Comment (agrojean-ledger) [reply tests/requirements.txt]: Removed

Comment (agrojean-ledger) [reply tests/requirements.txt]: Removed

**Review by fbeutin-ledger — APPROVED** [review]

**Pull request merged**